src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -17,10 +19,15 @@
     private TrelloClient trelloClient;
 
     @RequestMapping(method = RequestMethod.GET, value = "getTrelloBoards")
-    public void getTrelloBoards() {
+    private List<TrelloBoardDto> getTrelloBoards() {
 
         List<TrelloBoardDto> trelloBoards = trelloClient.getTrelloBoards();
-
-        trelloBoards.forEach(trelloBoardDto -> System.out.println(trelloBoardDto.getId() + " " + trelloBoardDto.getName()));
+        List<TrelloBoardDto> result = trelloBoards.stream()
+                .filter(trelloBoardDto -> trelloBoardDto.getName().contains("Kodilla"))
+                .filter(trelloBoardDto -> trelloBoardDto.getName() != null)
```

Comment:
Warunki powinny być w odwrotnej kolejności - jeśli trafi się `trelloBoard` bez `name` (czyli z `name` równym `null`), to w pierwszym filtrze poleci `NullPointerException` i całość się wysypie.